README.md

```
@@ -1,3 +1,160 @@
 
 장기 미션 저장소
+
+## 과제 진행 요구 사항
+
+첫 시작시의 장기의 보드판 그리고 각 기물을 구현하라
+게임 시작 시 전체 기물들과 장기판을 올바른 위치에 초기화 한다.
+
+## 기능 요구사항
+
+- [x] 기물(piece) 종류
+    - [x] 궁(宮): Palace (1개)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)